webapp/src/main/java/com/box/l10n/mojito/service/thirdparty/ThirdPartyTMSSmartlingGlossary.java

```diff
@@ -413,15 +459,27 @@ private VirtualAssetTextUnit mapGlossaryTermToVirtualAssetTextUnit(
   }
 
   private String getTextUnitComment(GlossarySourceTerm glossarySourceTerm) {
-    String comment = glossarySourceTerm.getDefinition();
+    StringBuilder commentBuilder = new StringBuilder();
+    commentBuilder.append(glossarySourceTerm.getDefinition());
     if (isPartOfSpeechAvailable(glossarySourceTerm)) {
-      comment =
-          glossarySourceTerm.getDefinition()
-              + " --- POS: "
-              + glossarySourceTerm.getPartOfSpeechCode();
+      commentBuilder.append(" --- POS: ").append(glossarySourceTerm.getPartOfSpeechCode());
+    }
+    if (glossarySourceTerm.getVariations() != null
+        && !glossarySourceTerm.getVariations().isEmpty()) {
+      commentBuilder.append(" --- Variations: ");
+      for (String variation : glossarySourceTerm.getVariations()) {
```

Comment:
I think this could be written in a more concise way: `String.join(", ", glossarySourceTerm.getVariations());`